app/boards/shields/bfo9000/bfo9000.yaml

```diff
@@ -0,0 +1,7 @@
+identifier: bfo9000
+name: BFO-9000
+type: shield
+manufacturer: keeb.io
+compatible: pro_micro
+components: [left, right]
```

Comment:
Minor: this syntax doesn't read or merge as easily as the list syntax.
e.g.
```
components:
   - left
   - right
```
I appreciate it'll probably make no difference as we're unlikely to produce merge conflicts on this line, but consistency throughout is usually a good thing.  A single syntax for arrays also gives beginners one less thing to worry about or get wrong for toolchain or compatible.